code/modules/mob/living/simple_animal/hostile/mining_mobs/hivelord.dm

```diff
@@ -411,14 +411,14 @@
 	)
 	var/type = pickweight(list(
 		"Miner" = 44,
-		"Waldo" = 3,
-		"Ashwalker" = 7,
-		"Soldier" = 3,
+		"SRM" = 10,
 		"Oldminer" = 8,
-		"Kobold" = 5,
+		"Ashwalker" = 7,
 		"Golem" = 6,
-		"SRM" = 10,
-		pick("Shadow", "YeOlde", "Operative", "Cultist") = 4
+		"Kobold" = 5,
+		"Soldier" = 3,
+		"Infiltrator" = 3,
```

Comment:
honestly I think it needs to be lower than even this, that's quite a lot of good syndie gear honestly